### ISSUE TITLE
Save Icon Check Box

### ISSUE THREAD

**Issue by Cuddles07** [ClusterM/hakchi2#814]
I noticed that adding the save icon to a game to indicate it has a save function is a variable in the desktop file. It would be nice to be able to have a check box next to the game title in the UI so we can enable this icon for games that support save functions (not save states)

An example

[X-CLOVER Game]
Code=CLV-U-BPHOF
TestID=777
ID=0
Players=2
Simultaneous=1
ReleaseDate=1995-11-20
SaveCount=0 <---This should be a 1 for a save function icon to appear in the menu next to the 2P icon
SortRawTitle=donkey kong country 2 - diddy's kong quest
SortRawPublisher=RARE
Copyright=hakchi2 ©2017 Alexey 'Cluster' Avdyukhin

**Comment by Cuddles07** [comment]
![example](https://user-images.githubusercontent.com/31124984/31583809-7962aa24-b156-11e7-86ca-618b40412bca.png)

As you can see here the icon is there after modifying that variable.

**Comment by naendow** [comment]
Indeed, that would be a great thing that would be very easy to add.

**Comment by ClusterM** [comment]
What's wrong with autodetection?

**Comment by Cuddles07** [comment]
Oh, I wasn't aware that auto detection was in. Still it would be nice for those that have already uploaded games to their SNES mini and don't want to have to re-add them.

**Comment by Drakonas** [comment]
But you can just load the game and let it save its data and presto. Takes literal seconds whereas it takes at least an hour or more to add a feature like this that barely anyone would use. Also, if you happen to erase your saves somehow, then the saved checkmarks would have to be micromanaged even more. To be honest this may just be too much micromanaging for most, and would get quite irritating to those that do not have a good memory. Kind of a moot point to request this. Closing.

Please do respond/reopen this issue if you can flesh out more reasons to why this should be added.

**Comment by mikezila** [comment]
Autodetection doesn't seem to work for emulators other than the inbuilt SNES one.

**Comment by Drakonas** [comment]
@mikezila please make a new ticket for this instead of responding to a week old ticket that is nonrelated to your problem.

**Comment by mikezila** [comment]
It's not unrelated, it's the same question/request.  Autodetection of the save capability badge doesn't seem to work with emulators other than the built-in SNES one.  It doesn't detect the save capability on the mGBA core or the genesis-plus core.  I've tested loading a GBA/Genesis game and making a battery save, and the icon with the cartridge and pencil is still missing.

It would be nice to have a checkbox in the UI to enable this icon for systems that do not support autodetection.

**Comment by Drakonas** [comment]
@mikezila sorry I neglected to realize you were responding to my last message. I apologize. I reacted too fast before realizing what you were talking about. I have closed about 200 tickets in the last few weeks. :P

**Comment by Drakonas** [comment]
@mikezila my point was to make a separate ticket requesting the bug for autodetection to be fixed for retroarch instead of requesting this workaround to be implemented. That is the source of the issue.
